sdks/java/core/src/main/java/org/apache/beam/sdk/testing/ValueInSingleWindow.java

```diff
@@ -109,7 +109,7 @@ public void encode(ValueInSingleWindow<T> windowedElem, OutputStream outStream,
       T value = valueCoder.decode(inStream, nestedContext);
       Instant timestamp = InstantCoder.of().decode(inStream, nestedContext);
       BoundedWindow window = windowCoder.decode(inStream, nestedContext);
-      PaneInfo pane = PaneInfo.PaneInfoCoder.INSTANCE.decode(inStream, nestedContext);
+      PaneInfo pane = PaneInfo.PaneInfoCoder.INSTANCE.decode(inStream, context);
```

Comment:
Anytime we have a case where we know that all but one of the coders are not context sensitive, we should always place the context sensitive coder to be last.